### DISCUSSION TITLE
用的是Alibaba Cloud Linux 3，安装失败！密码无法更改，应用也无法启动！

### DISCUSSION THREAD

**Discussion by hack0573** [AlistGo/alist#5908] [General]
下载 Alist latest ...
####################################                                      50.7%
curl: (18) transfer closed with 12005258 bytes remaining to read

gzip: stdin: unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
 下载成功 
Alist 安装成功！

访问地址：http://YOUR_IP:5244/

配置文件路径：/home/alist/data/config.json
---------如何获取密码？--------
先cd到alist所在目录:
cd /home/alist
随机设置新密码:
./alist admin random
或者手动设置新密码:
./alist admin set NEW_PASSWORD
----------------------------
启动服务中

查看状态：systemctl status alist
启动服务：systemctl start alist
重启服务：systemctl restart alist
停止服务：systemctl stop alist

温馨提示：如果端口无法正常访问，请检查 服务器安全组、本机防火墙、Alist状态

[root@hqwaliyun2024 home]# ^C
[root@hqwaliyun2024 home]# pwd
/home
[root@hqwaliyun2024 home]# cd alist/
[root@hqwaliyun2024 alist]# ll
total 32928
-rwx------ 1 root root 33718272 Jan 19 11:23 alist
[root@hqwaliyun2024 alist]# ./alist admin set xxxxxx
Segmentation fault
[root@hqwaliyun2024 alist]# ./alist admin random
Segmentation fault
[root@hqwaliyun2024 alist]# ^C
[root@hqwaliyun2024 alist]# systemctl status alist
● alist.service - Alist service
   Loaded: loaded (/etc/systemd/system/alist.service; enabled; vendor preset: enabled)
   Active: failed (Result: signal) since Fri 2024-01-19 11:23:26 CST; 12min ago
  Process: 24248 ExecStart=/home/alist/alist server (code=killed, signal=SEGV)
 Main PID: 24248 (code=killed, signal=SEGV)

Jan 19 11:23:26 hqwaliyun2024 systemd[1]: Started Alist service.
Jan 19 11:23:26 hqwaliyun2024 systemd[1]: alist.service: Main process exited, code=killed, status=1>
Jan 19 11:23:26 hqwaliyun2024 systemd[1]: alist.service: Failed with result 'signal'.

[root@hqwaliyun2024 alist]# systemctl start alist
[root@hqwaliyun2024 alist]# systemctl status alist
● alist.service - Alist service
   Loaded: loaded (/etc/systemd/system/alist.service; enabled; vendor preset: enabled)
   Active: failed (Result: signal) since Fri 2024-01-19 11:36:08 CST; 3s ago
  Process: 24304 ExecStart=/home/alist/alist server (code=killed, signal=SEGV)
 Main PID: 24304 (code=killed, signal=SEGV)

**Comment by ititchuan** [comment]
这个问题解决了吗？我也是这个问题。
